whois-nrtm4-client/src/test/resources/mock/nrtm-delta.2.RIPE-NONAUTH.json

```diff
@@ -0,0 +1,3 @@
+{"nrtm_version":4,"type":"delta","source":"RIPE-NONAUTH","session_id":"6328095e-7d46-415b-9333-8f2ae274b7c8","version":2}
+{"action":"delete","object_class":"ROUTE6","primary_key":"2001:490:c000::/35AS18666"}
```

Comment:
Primary key is coming as 
2001:490:c000::/35 in our NRTM service
I think that the AS number should be attached to it "2001:490:c000::/35AS18666"
as I show in this mock.
We have the problem that the prefix of a route can exist for different AS in another RIR. That's why we added the AS number at the end, I think that here we should do the same

Reply:
Created a task for fix this